app/src/main/java/com/safeboda/commons/analytics/entity/PropertyValues.kt

```diff
@@ -2,4 +2,10 @@ package com.safeboda.commons.analytics.entity
 
 internal const val USER_ID = "id"
 internal const val USER_IDENTIFIER = "identifier"
-internal const val IS_USER_LOGGED_IN = "is_user_logged_in"
\ No newline at end of file
+internal const val IS_USER_LOGGED_IN = "is_user_logged_in"
+
+// CleverTap pre-defined profile properties
+internal const val USER_NAME = "Name"
```

Comment:
Shouldn't be in lower-case, like above ones?

Reply:
Following the CleverTap docs, these are a pre-defined values so we need to send them exactly like this.